src/locale/messages.xlf

```diff
@@ -1297,14 +1298,6 @@
         <source>Account Settings</source>
       </segment>
     </unit>
-    <unit id="workspace.developer_tools">
```

Comment:
Is renoving this ok?

Reply:
Yes since is not been removed, just moved into line 152.